gitleaves/reports.py

```diff
@@ -68,16 +72,29 @@ def get_next_leaves_by_month(bydates):
     return nextbymonths
 
 
-def export_month_csv(year, month):
-    leaves_csv_path = leaves_csv_path_pat.format(YYYY=year)
-    data = load_csv(leaves_csv_path)
-    outfile = f'/tmp/leaves.{year}.{month}.csv'
-    outdata = ((d.isoformat(), *names)
-               for d, names in
-               data['bymonths'][month].items())
-    csv.writer(open(outfile, 'w')).writerows(outdata)
-    print(f'Data exported to {outfile}')
+# Shekhar’s original in case Jason messes up, — mjb 12/20/2021
+# def export_month_csv(year, month):
+#     leaves_csv_path = leaves_csv_path_pat.format(YYYY=year)
+#     data = load_csv(leaves_csv_path)
+#     outfile = f'/tmp/leaves.{year}.{month}.csv'
+#     outdata = ((d.isoformat(), *names)
+#                for d, names in
+#                data['bymonths'][month].items())
+#     csv.writer(open(outfile, 'w')).writerows(outdata)
+#     print(f'Data exported to {outfile}')
```

Comment:
I understood why you have chosen to do keep commented out code but it is not needed. 
Git does this for us. Infact this has hindered the diff view

Reply:
Shoot! this was just temporary while i worked on this. It *was* supposed to come out before the pr 😂
Removed.